servers/rendering/renderer_rd/shaders/environment/sdfgi_debug.glsl

```diff
@@ -37,16 +37,13 @@ layout(push_constant, std430) uniform Params {
 	uint max_cascades;
 
 	ivec2 screen_size;
-	bool use_occlusion;
 	float y_mult;
 
-	int probe_axis_size;
 	float z_near;
-	float reserved1;
-	float reserved2;
 
-	mat4 cam_transform;
-	mat4 inv_projection;
+	float inv_projection[4][3];
+	float cam_basis[3][3];
```

Comment:
This could be a mat3 to save creating one

Reply:
std430 rules would require to add padding. Maybe there's a way to re-order the members so that, even considering alignment rules, we can stay within size, but I haven't found.